packages/config/config/devices/0x0179/ch-103.json

```diff
@@ -0,0 +1,176 @@
+{
+	"manufacturer": "ConnectHome",
+	"manufacturerId": "0x0179",
+	"label": "CH-103",
+	"description": "CH-103 Relay",
+	"devices": [
+		{
+			"productType": "0x0041",
+			"productId": "0x0014"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Target for temperature and alarm reports",
+			"maxNodes": 10,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Automatic Calibration",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Normal",
+					"value": 0
+				},
+				{
+					"label": "Calibration",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Operation Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Standard",
+					"value": 0
+				},
+				{
+					"label": "Motor with limit switchs",
+					"value": 1
+				},
+				{
+					"label": "Blind Control",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "Rotation Time",
+			"description": "Time of full rotation of lamellas or (180 °) in the blind mode",
+			"unit": "decisecond",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 0
+		},
+		{
+			"#": "4",
+			"label": "Open Operating Time",
+			"unit": "decisecond",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 0
+		},
+		{
+			"#": "5",
+			"label": "Close Operating Time",
+			"unit": "decisecond",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 0
+		},
+		{
+			"#": "6",
+			"label": "Open Delay",
+			"unit": "decisecond",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 0
+		},
+		{
+			"#": "7",
+			"label": "Close Delay",
+			"unit": "decisecond",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 0
+		},
+		{
+			"#": "50",
+			"label": "Response Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Security and regular commands",
+					"value": 0
+				},
+				{
+					"label": "Security commands only",
+					"value": 1
+				}
+			]
```

Comment:
I don't understand this parameter. Maybe the label can be improved or a short description added that helps?

Reply:
Here is what Edge translates this to.   So it looks like a security command class vs some other command class. 

`
Module CH-103 - supports the transmission of radio commands in safe mode (Security Command Class). Security Command Class is the transmission of control commands using hardware encryption over the 128-bit AES algorithm. This allows you to completely protect the device from unauthorized control. Refer to your controller's manual for support for safe mode devices.

0 - When you click on Switch 1, send the usual command to the 2nd group of links (Default);; 1 - When you press switch 1, send a secure command to the 2nd group of connections.`

Reply:
Oh, so whether the device should accept non-encrypted commands, even if it was included securely.
Lets do:
- label: `"Respond to Non-Encrypted Commands When Included Securely"`
- options: `Enable` (0), `Disable` (1)

Side note: The fact that this is even an option is bothersome. But since the default value of accepting the non-secure commands is even more stupid, we need to document this parameter, so users can turn it off.